lib/MetaCPAN/Web/Controller/Author.pm

```diff
@@ -50,15 +51,15 @@ sub index : Chained('root') PathPart('') Args(0) {
     my ( $author, $data ) = ( $author_cv->recv, $releases_cv->recv );
     $c->detach('/not_found') unless ( $author->{pauseid} );
 
-    my $faves_cv = $c->model('API::Favorite')->by_user( $author->{user} );
+    my $faves_cv = $c->model('API::Favorite')->by_user( $author->{pauseid} );
```

Comment:
Ah, ok.  It should be there.  It's there on production.  http://api.metacpan.org/author/OALDERS?fields=user